modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/toolbar/contributor/helper/MenuItemProvider.java

```diff
@@ -66,6 +66,18 @@
 @Component(service = MenuItemProvider.class)
 public class MenuItemProvider {
 
+	public static String getIcon(DLFileEntryType fileEntryType) {
```

Comment:
@GabrielHenriquedev This is inlined from DLFileEntryTypeIconProviderUtil since that util is only used here. So after inline, the method shouldn't be kept `public static`. Just make it `private`.